systems/simple_test.go

```diff
@@ -0,0 +1,12 @@
+package systems
+
+
+import "testing"
+
+func TestGetMemoryUsage(t *testing.T) {
+      c := new(LocalSystem)
+    Memory := c.GetMemoryUsage()
```

Comment:
This local variable should use a lowercase `m` for the first letter. Please remove all unnecessary spacing